test/module/iroha-cli/client_test.cpp

```diff
@@ -151,3 +152,125 @@ TEST_F(ClientTest, SendTxWhenStatelessInvalid) {
   std::cout << "Sending  json transaction to Iroha" << std::endl;
   ASSERT_EQ(client.sendTx(json_tx), iroha_cli::CliClient::NOT_VALID);
 }
+
+TEST_F(ClientTest, SendQueryWhenInvalidJson) {
+  iroha_cli::CliClient client(Ip, Port);
+  // Must not call stateful validation
```

Comment:
Also, confusing comment. Because not provide new information `EXPECT_CALL(...).Times(0)`, but if is it may be write reason, why it is must to not call?